src/pages/styles/typography.tsx

```diff
@@ -52,9 +52,9 @@ const Page = (): JSX.Element => (
         <SEO title={t('title')} />
         <Heading.h1>{t('title')}</Heading.h1>
 
-        <Paragraph.lead>
+        <Paragraph>
```

Comment:
Why are these (in this and other files) `Paragraph.lead` changed to just `Paragraph`?
Because it seems that the `suomifi-design-system/src/components/Paragraph.tsx` in unchanged..?

Reply:
My bad, removed those because linter complains about those - which needs to be fixed later on. Added those back.